### DISCUSSION TITLE
Cursors to extract data points in in graphs

### DISCUSSION THREAD

**Discussion by bnilsson11** [ra3xdh/qucs_s#890] [Q&A] [answered]
It is sometimes useful to extract a specific numerical value from a graph, but this is difficult (unless I have missed it).
One solution would be to export as CSV, but this a bit complicated to do in this context, more work for a single value
In the PCB application KiCad which is also using ngspice, the sim display have cursors to sample points in the graph, works fine.
However, since qucs-s is more convenient to use for simulations, I would appreciate if cursor were available here as well.
Is this possible to implement?

**Comment by ra3xdh** [comment]
The existing marker properties dialog shows only X-axis position. I can add a possibility to show Y-axis marker value. The user can copy-paste the both values. I mean the following dialog window:
![image](https://github.com/user-attachments/assets/c8326d0d-7f2e-43dc-ba0d-3fc595039458)

**Comment by bnilsson11** [comment, marked as answer]
Indeed I have missed something here.
I have now found "set marker on graph", and it seems to do what I want, no need to extend.
Thanks.
